config/index.js

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash';
+
+import variantBase from './variants/_base';
+
+export const VARIANTS = [
+  'development',
+  'demo',
+  'production',
```

Comment:
I'm mildly concerned about confusion between `VARIANTS` and `NODE_ENV`. How would you distinguish the two?